libpod/container.go

```diff
@@ -407,6 +409,58 @@ func (c *Container) teardownStorage() error {
 	return nil
 }
 
+func (c *Container) addImageData(g *generate.Generator) (err error) {
```

Comment:
Is collision handled by timing? meaning if something is set by the image, it will get set but then user input will override it ?

Reply:
I think we need to explicitly check the spec and config we got from the user, and not override if we see things being set.

Reply:
Yes the way this should be is the user data should override the image data.